### PR TITLE
proof of concept for chrome extension

### PR DESCRIPTION
**Pull request by ilya**
Steps to run:
1. start servers
2. ./cclient proxy
3. In Chrome, go to chome://extensions -> load unpacked extension -> navigate to chrome/ dir. Then click the new toolbar button.

### CODE REVIEW

**Comment by ilya** [comment]
@lacker any commentary? I think we/you probably need to decide where to home the HTTP API before any more real progress should be made. The approach in this PoC isn't going to scale well as the API surface area increases.

File: cmd/cclient/main.go [hunk 1]
```diff
@@ -86,10 +89,28 @@ func send(recipient string, amountStr string) {
 	log.Printf("transaction %d cleared", transaction.Sequence)
 }
 
+func handler(w http.ResponseWriter, r *http.Request) {
+	pass :=strings.TrimLeft(r.URL.Path, "/")
+	kp := util.NewKeyPairFromSecretPhrase(pass)
+	s := status(kp.PublicKey())
+	if s != nil {
+		fmt.Fprintf(w, "{ \"sequence\": %v, \"balance\": %v }",
```
Comment (lacker) [inline cmd/cclient/main.go]: i think you want the %v here to be more specific, like %d

File: cmd/cclient/main.go [hunk 2]
```diff
@@ -86,10 +89,28 @@ func send(recipient string, amountStr string) {
 	log.Printf("transaction %d cleared", transaction.Sequence)
 }
 
+func handler(w http.ResponseWriter, r *http.Request) {
+	pass :=strings.TrimLeft(r.URL.Path, "/")
```
Comment (lacker) [inline cmd/cclient/main.go]: space after := is missing, maybe check that go fmt ran?

**Comment by lacker** [comment]
Hmm, well the code looks good, and you are right that it needs to be more clear what sort of HTTP API is useful here. I should probably get a testnet running - there might not even be a need for a local proxy.

Can you add like a README in this directory or something to explain how to install/run the chrome extension?

File: cmd/cclient/main.go (hunk 2 above)
Comment (ilya) [reply cmd/cclient/main.go]: I'll run it. Is it supposed to run automatically or something?

File: cmd/cclient/main.go (hunk 1 above)
Comment (ilya) [reply cmd/cclient/main.go]: Will do.

**Comment by ilya** [comment]
Yep, sure thing. How would a testnet solve the HTTP issue? Don't you need to convert away from raw sockets first?

**Comment by ilya** [comment]
All comments addressed.

**Comment by lacker** [comment]
My statement was a non sequitur - a testnet wouldn't solve the http issue, but this pull request is making it more clear that a testnet would be useful.

**Comment by ilya** [comment]
Ah, yeah, indeed. Alright, LMK if you think anything else worth doing in this PR until then. Otherwise will hunt around for some other TODOs.

**Pull request merged**